app/src/main/java/com/ciandt/dragonfly/example/features/projectselection/ProjectSelectionActivity.kt

```diff
@@ -134,13 +143,17 @@ class ModelSelectionActivity : BaseActivity(), ModelSelectionContract.View {
         startActivity(intent)
     }
 
-    override fun showDownloading(model: Model) {
-        Snackbar.make(getRootView(), getString(R.string.model_selection_item_wait), Snackbar.LENGTH_LONG).show()
+    override fun showDownloading(project: Project) {
+        Snackbar.make(getRootView(), getString(R.string.project_selection_item_wait_message), Snackbar.LENGTH_LONG).show()
```

Comment:
What do you thinking of adding extensions to Activity (showShortSnackbar() / showLongSnackbar)? Or just something that pass getRootView() by default?